provisioning/plugins/corim-psa-decoder/extractor.go

```diff
@@ -6,14 +6,33 @@ import (
 	"errors"
 	"fmt"
 
-	"github.com/veraison/services/proto"
 	"github.com/veraison/corim/comid"
+	"github.com/veraison/services/proto"
 	structpb "google.golang.org/protobuf/types/known/structpb"
 )
 
-type Extractor struct{}
+const (
+	cca_profile = "http://arm.com/cca/ssd/1"
+)
+
+type Extractor struct {
+	Profile string
+}
+
+func (o Extractor) GetProfile() string {
+	return o.Profile
+}
+
+func (o *Extractor) SetProfile(p string) {
+	o.Profile = p
+}
 
-func (o Extractor) SwCompExtractor(rv comid.ReferenceValue) ([]*proto.Endorsement, error) {
+type MeasExtractor interface {
```

Comment:
should be called RefValExtractor

Reply:
RefValExtractor is already taken when extratcing refVal from CoRIM.

Here we are extracting Measurements from Comid so I named it as MeasExtractor

Reply:
ok, let's call it `MeasurementExtractor` then -- there is no need to worry about the number of characters :-)